packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1101,9 +1101,18 @@ typedef void PointerMoveEventListener(PointerMoveEvent event);
 typedef void PointerUpEventListener(PointerUpEvent event);
 typedef void PointerCancelEventListener(PointerCancelEvent event);
 
+/// How to behave during hit tests.
 enum HitTestBehavior {
+  /// Consider this target hit if, and only if, its child was hit.
   deferToChild,
+
+  /// Consider this target hit whenever the position being hit tested is within its bounds.
   opaque,
+
+  /// Never consider this target hit but always add the target to the event path anyway.
```

Comment:
That's a confusing way to put it. Isn't "hit" the same as "put it in the event path"?

How about "Consider this target hit whenever the position being hit tested is within its bounds, and continue hit testing further elements behind this one."

And maybe the other two should be rephrased to split the "getting hit" concept from the "prevent going through" concept? I'm not sure exactly how to phrase this.

Maybe just remove the "Never consider..." sentence and just let the other sentence for translucent describe it...